src/Api/Clips.php

```diff
@@ -39,7 +40,7 @@ public function getClip($slug)
      * @throws UnsupportedOptionException
      * @return array|json
      */
-    public function getTopClips($channel = null, $cursor = null, $game = null, $limit = 10, $period = 'day', $trending = false)
+    public function getTopClips($channel = null, $cursor = null, $game = null, $language = null, $limit = 10, $period = 'day', $trending = false)
```

Comment:
I'd be concerned with adding a new parameter into the middle of the argument list. This will break any existing calls of this method where people expect the fourth parameter to still be limit. If this is accepted and merged as is, it should be made clear somehow that this breaking change was made. Not sure how to best accomplish that.

Reply:
Just wanted to have it in the same order as in the clips api documentation. But i can change that if a major version would be a problem

Reply:
I'll let @nicklaw5 make the final decision. I just saw it and wanted to comment on it.

Reply:
> If this is accepted and merged as is, it should be made clear somehow that this breaking change was made. Not sure how to best accomplish that.

@echosa This is one of the reasons I was hoping to keep v1.x and v2.x separated. It makes it difficult to bump versions, with both under the same branch.

I'm all for accommodating breaking changes but this is too small a change to introduce one. @zlokomatic do you mind shifting the new parameter to the end?

Reply:
Moved the parameter